packages/nodejs/test/registry.ts

```diff
@@ -0,0 +1,53 @@
+import { Span, SpanOptions, SpanContext, SpanData } from "../src/interfaces"
+import { RootSpan, ChildSpan } from "../src/span"
+
+export class TestRegistry {
+  static spans: Span[] = []
+  static closedSpans: Span[] = []
+
+  static addSpan(span: Span) {
+    this.spans.push(span)
+  }
+
+  static addClosedChildSpan(span: Span) {
```

Comment:
This method says `ChildSpan`, but the other ones just say `Span`. Is there a situation where `addSpan` could be called with a `RootSpan`? If not, then I'd pick one term or the other.

Reply:
Yes. I'm renaming things to just support both types of spans.